src/app/layouts/components/header/header.component.scss

```diff
@@ -3,25 +3,25 @@
 
 .header {
     background: $white;
-    padding: 10px;
+    padding: 3px;
     width: 100%;
     display: flex;
-    font-size: 30px;
+    font-size: 11px;
```

Comment:
you should define variables for default font sizes like `small`, `medium`, `large` and try to use those in order to use the same sizes everywhere